apps/wappush/js/cp_screen_helper.js

```diff
@@ -455,7 +452,7 @@ var CpScreenHelper = (function() {
                                                      authInfo);
       } catch (e) {
         message = finishConfirmDialog.querySelector('strong');
```

Comment:
I'd put "var message" here instead of having it at the top of the function -- makes it easier to understand it's used only here.